sdk/storage/Azure.Storage.Common/swagger/Generator/src/serviceModel.ts

```diff
@@ -611,7 +611,8 @@ function createResponse(project: IProject, code: string, name: string, swagger:
         bodyClientName: <string>optional(() => swagger[`x-az-response-schema-name`], `Body`), // TODO: switch from 'Body' to body.name?
         headers,
         exception: <boolean>optional(() => swagger[`x-az-create-exception`]),
-        public: isPublic
+        public: isPublic,
+        returnStream: <boolean>optional(() => swagger[`x-az-stream`])
```

Comment:
Please add this to the list in https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Common/swagger/Generator/readme.md so we can keep track of all the extensions.